nannou/src/color/mod.rs

```diff
@@ -105,6 +106,15 @@ where
     LinSrgba::new(r, g, b, a)
 }
 
+/// Create a new color from a hexadecimal int literal
+#[inline]
+pub fn rgb_u32(c: u32) -> color::Color {
+    let blue: u8 = (c & 0xFF) as u8;
+    let green: u8 = ((c >> 8) & 0xFF) as u8;
+    let red: u8 = ((c >> 16) & 0xFF) as u8;
+    color::rgb_bytes(red, green, blue)
```

Comment:
Would you mind using the `rgb8` function that is in this module, rather than the `ui::color::rgb_bytes` function here? E.g.
```rust
rgb8(red, green, blue)
```
This should also allow for removing the `use crate::ui::color` import. You may also need to change the return type to `Rgb<u8>`.

Reply:
No problem, just pushed a change that should fix this 😀